arangod/Replication/Syncer.cpp

```diff
@@ -341,6 +351,7 @@ void Syncer::JobSynchronizer::jobDone() {
 
   TRI_ASSERT(_jobsInFlight == 1);
   --_jobsInFlight;
+  _condition.signal();
```

Comment:
we probably also need a way to signal the JobSynchronizer in the `setAborted(bool)` method ?